README.md

```diff
@@ -1 +1,184 @@
+- [Hugo](https://gohugo.io/getting-started/installing/)
+
+If you're running the site for the first time, or if dependencies have changed, install dependencies with:
+
+`yarn install`
+
+### Pre-load Webpack assets
+
+Additionally, you may want to run:
+
+`npm run build:webpack`
+
+Hugo is so fast that the first time you run the site, the Hugo build will happen much more quickly than the webpack build so it's best to run the build once so that assets are ready for when you start the site.  If you don't see any styling the first time you visit the page, force refresh it.  If that doesn't work, try restarting the site.
+
+### External API's
+
+This site requires running instances of [`ocw-studio`](https://github.com/mitodl/ocw-studio) and [`open-discussions`](https://github.com/mitodl/open-discussions) for some functionality.  Search results are provided by `open-discusisons` and `ocw-studio` provides some content for the home page, such as newly added courses and news items.  If you need to work with this functionality you can either run a local instance of either of these projects, or alternatively point at the RC instances and use a browser extension to temporarily disable CORS.
+
+### CORS
+
+The search page at `/search` uses the `open-discussions` search API to source results.  Running this locally and populating it with results can be tedious, so it's often easier to just point your local website at an already running version of the search API.  In order for this to work properly, you will need to use a browser extension for disabling CORS, like Moesif Origin & CORS Changer:
+
+- [Chrome](https://chrome.google.com/webstore/detail/moesif-origin-cors-change/digfbfaphojjndkpccljibejjbppifbc?hl=en-US#:~:text=Moesif%20Origin%20%26%20CORS%20Changer&text=This%20plugin%20allows%20you%20to%20send%20cross%2Ddomain%20requests.&text=This%20plugin%20allows%20you%20to%20send%20cross%2Ddomain%20requests%20directly,Allow%2DOrigin%20set%20to%20*.)
+- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/moesif-origin-cors-changer1/)
+### Environment variables
+
+For development you'll need to set a few environment variables. The
+`.env.example` file will get you started.  Here are the variables and what they do:
+
+| Variable | Example | Description |
+| --- | --- | --- |
+| `SEARCH_API_URL` | `http://localhost:8063/api/v0/search/` | A URL to an `open-discussions` search API to fetch results from|
+| `OCW_STUDIO_BASE_URL` | `http://localhost:8043/` | A URL of an instance of [`ocw-studio`](https://github.com/mitodl/ocw-studio) to fetch home page content from |
+| `EXTERNAL_SITE_PATH` | `~/Code/ocw-www/site/` | A path to a Hugo site with content (Pages, Notifications, Promos, Testimonials) to use for local development |
+| `OCW_TO_HUGO_PATH` | `~/Code/ocw-to-hugo/` | A path to a local override of [`ocw-to-hugo`](https://github.com/mitodl/ocw-to-hugo), a library used to generate Hugo markdown content from parsed OCW JSON exports from the legacy site |
```

Comment:
Why does the theme depend on ocw-to-hugo?

Reply:
When running a course site locally, if you specify a path to `ocw-to-hugo`, the `start_course.sh` script will automatically download the parsed JSON for the course you specify and convert it to markdown to spin up the site with.  It's not necessarily required, but is a quick way to get up and running if (as a developer) you're trying to work on changes to the theme and see them reflected on an existing course.  This will change of course once we moonlight `ocw-to-hugo`.  You can still use the course theme in an external folder of content by including it in your `config.toml`, and we should add a way to do that here without using `ocw-to-hugo`.